training/provision/playbook.yml

```diff
@@ -15,16 +15,51 @@
   - name: Gather facts for first time
     ansible.builtin.setup:
 
-  - name: Check Podman Present
-    ansible.builtin.package:
-      name: podman
+  - name: Required packages
+    ansible.builtin.dnf:
+      name:
+      - https://s3.eu-west-2.amazonaws.com/amazon-ssm-eu-west-2/latest/linux_amd64/amazon-ssm-agent.rpm
+      - podman
       state: present
+      disable_gpg_check: true
 
-  - name: Bootc install
+  - name: Derived Image Containerfile
+    ansible.builtin.template:
+      src: ./templates/Containerfile.j2
+      dest: /tmp/Containerfile
```

Comment:
Minor nit but, predictable filenames in `/tmp` are a CWE: https://capec.mitre.org/data/definitions/149.html

There's a [tempfile module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/tempfile_module.html).

Not a big deal, but just noting.